### PR TITLE
chore: 3.8.x

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

File: helpers/compile/build.ts [hunk 1]
```diff
@@ -4,14 +4,15 @@ import { flatten } from '../blaze/flatten'
 import { pipe } from '../blaze/pipe'
 import { map } from '../blaze/map'
 import { transduce } from '../blaze/transduce'
-import glob from 'glob'
+import glob from 'globby'
```
Comment (janpio) [inline helpers/compile/build.ts:7]: Was this import broken before?

File: helpers/compile/plugins/depCheckPlugin.ts [hunk 2]
```diff
@@ -94,25 +90,13 @@ const unusedPlugin: esbuild.Plugin = {
...
-      ignore: ['**/packages/**/*', '**/*.d.ts'],
-      gitignore: true,
-    }),
-    logLevel: 'silent', // there will be errors
-    bundle: true, // we bundle to get everything
-    write: false, // no need to write for analysis
-    outdir: 'out',
-    plugins: [unusedPlugin],
-  })
-  .catch(() => {})
```
Comment (janpio) [inline helpers/compile/plugins/depCheckPlugin.ts:118]: Where does this logic go?

File: packages/debug/package.json [hunk 3]
```diff
@@ -52,7 +50,8 @@
   ],
   "dependencies": {
     "@types/debug": "4.1.7",
-    "ms": "2.1.3"
+    "ms": "2.1.3",
+    "strip-ansi": "6.0.1"
```
Comment (janpio) [inline packages/debug/package.json:54]: Is this really a normal dep? Where is it used, or waht for?

File: packages/migrate/package.json [hunk 4]
```diff
@@ -57,12 +54,16 @@
...
+    "chalk": "4.1.2",
     "execa": "5.1.1",
     "has-yarn": "2.1.0",
     "indent-string": "4.0.0",
     "log-update": "4.0.0",
+    "mariadb": "2.5.5",
+    "mssql": "7.3.0",
     "new-github-issue-url": "0.2.1",
     "open": "7.4.2",
+    "pg": "8.7.1",
```
Comment (janpio) [inline packages/migrate/package.json:66]: Are these really used outside of tests?

Comment (millsp) [reply packages/migrate/package.json:66]: Same as https://github.com/prisma/prisma/pull/11156#discussion_r784860441

File: helpers/compile/plugins/depCheckPlugin.ts (hunk 2 above)
Comment (millsp) [reply helpers/compile/plugins/depCheckPlugin.ts:118]: It moved to `prisma/helpers/compile/build.ts`

File: helpers/compile/build.ts (hunk 1 above)
Comment (millsp) [reply helpers/compile/build.ts:7]: Although they are compatible, `globby` had a gitignore option. Thinking about it, it is not even needed anymore, so I could revert that in a later PR.

**Pull request closed without merging**